AppWidget/glance-widget/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (C) 2021 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
```

Comment:
I think that is the case where you want to set match_parent.
Setting the width and height to 0dp doesn't result in an expected size for wrap_content.

Reply:
BTW, why don't you just set width and height to `match_parent` here? It seems that's what you want anyway.

Reply:
Because there will be no space other than this TextView if match_parent is specified. (this TextView is the only view in this activity, at the moment though). 

I think it's not needed to set the match_parent to a view, which shouldn't take the entire space. Instead centering a view, which has the desired size for that view makes more sense.